src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = days * 40;
+
+  if (days >= 7) {
+    total = total - 50;
+  } else if (days >= 3) {
```

Comment:
Consider using a constant for the discount for rentals of 3 or more days (20) to make the code more maintainable and easier to understand. E.g. `const threeDayDiscount = 20;`